### DISCUSSION TITLE
Probable  release date of Jedis version 4.4.0

### DISCUSSION THREAD

**Discussion by Datta19** [xetorthio/jedis#3384] [Q&A] [answered]
Is anybody have any idea when Jedis version 4.4.0 will be released. I confirmed that from Jedis version 4.4.0, UnifiedJedis will have a multi() support.

**Comment by sazzad16** [comment, marked as answer]
4.4.0-rc1 is out. Unless there are unexpected circumstances, 4.4.0 should be released _soon_.

**Reply by Datta19** [reply]
@sazzad16 Thanks for the quick response.
